cmd/collectors/restperf/restperf.go

```diff
@@ -504,14 +504,20 @@ func (r *RestPerf) PollData() (map[string]*matrix.Matrix, error) {
 				return true
 			}
 
-			// extract instance key(s)
-			for _, k := range instanceKeys {
-				value := parseProperties(instanceData, k)
-				if value.Exists() {
-					instanceKey += value.String()
-				} else {
-					r.Logger.Warn().Str("key", k).Msg("skip instance, missing key")
-					break
+			if len(instanceKeys) != 0 {
```

Comment:
is this guard clause needed? range handles empty and nil slice

Reply:
Agreed that range would handle nil and empty. But here, we need to isolate these 2 cases,
1. instancekeys not specified --> `cluster scoped templates` --> here we create instance with empty("") instanceKey and proceed as expected.
2. instancekeys specified but got empty response from ONTAP --> here we log and skip that instance.